src/System.Text.Json/tests/JsonDateTimeTestData.cs

```diff
@@ -229,6 +229,9 @@ public static IEnumerable<object[]> InvalidISO8601Tests()
             yield return new object[] { "\"199\\u0037-07\\u002d16T1\\u0039:20:30.4555555+\\u002dZ\"" };
             // Proper format but invalid calendar date, time, or time zone designator fields 1997-00-16
             yield return new object[] { "\"\\u0031\\u0039\\u0039\\u0037\\u002d\\u0030\\u0030\\u002d\\u0031\\u0036\"" };
+
+            // High byte expansion
```

Comment:
Let's add some valid date time tests that use escaped characters as well.

Reply:
You mean like this?

https://github.com/dotnet/corefx/blob/70150e6750c35c9a67803e9cddfda7b1db2ec487/src/System.Text.Json/tests/JsonDateTimeTestData.cs#L84

Reply:
Yep :)
Do we use those for `JsonString.GetDateTime`?